docs/data-scientists-root.md

```diff
@@ -1,49 +1,84 @@
+
 - channel
 - minimum version
 - country
 - locale
-- other?
+- OS
+- client preference values (on desktop)
+- other factors
+
+This additional filtering always happens logically _after_ a client passes the sizing filter.
+You must inflate your population fraction to account for filtering.
+
+For a concrete example, imagine that Firefox WAU is 1,000 clients. 20% of WAU is from Canada. You wish to deploy an experiment to Canadian users. Your power analysis says that you need 50 clients in total to enroll. You should specify a population fraction of at least 25%, because 1,000 * 0.2 (from Canada) * 0.25 (your filter) = 50.
 
-The above attributes are the only ones that Nimbus is able to filter to on, before the beginning of enrollment. If you desire additional filters (e.g., filters on preference settings), then Nimbus will apply those filters as clients enroll in the experiment. However, since Nimbus is not able to apply additional filters before the beginning of enrollment, you must inflate your sample size recommendations assuming that clients not satisfying additional filters will be included in the population.
+Most of our telemetry metrics are not normally distributed. A couple approaches that you may find helpful are:
 
-Many of our telemetry metrics are not normally distributed. For non-normally distributed data, you may consider a Mann-Whitney U-test to calculate sample sizes for a comparing 2+ samples. [Gpower](https://www.psychologie.hhu.de/arbeitsgruppen/allgemeine-psychologie-und-arbeitspsychologie/gpower) implements the Mann-Whitney U-test.
+* powering a Mann-Whitney U-test. [Gpower](https://www.psychologie.hhu.de/arbeitsgruppen/allgemeine-psychologie-und-arbeitspsychologie/gpower) implements the Mann-Whitney U-test.
+* log-transforming the data and the expected difference and powering a _t_ test in log space.
 
 ## Metrics and statistics
 
-Two weeks after the beginning of the enrollment period, Jetstream will begin to produce auto-generated reports summarizing the results of the experiment. [Here](https://experimenter.services.mozilla.com/nimbus/custom-messaging-in-aboutwelcome-for-chrome-users-to-import/results) is an example of that.
+Two weeks after the beginning of the enrollment period, Jetstream will begin to produce auto-generated reports summarizing the results of the experiment. [Here is one such report](https://experimenter.services.mozilla.com/nimbus/custom-messaging-in-aboutwelcome-for-chrome-users-to-import/results) (internal link).
 
-To see which metrics are included by default to this auto-generated report as well as information on adding custom and default metrics, statistics and confidence, see the [Jetstream documentation](https://experimenter.info/jetstream/jetstream).
+To see which metrics are included by default to this auto-generated report as well as information on adding custom and default metrics, statistics and confidence, see the [Jetstream documentation](jetstream/jetstream.md).
```

Comment:
Same here. I think changing this extension will cause it to 404 since the jetstream.md file will be converted into something like https://experimenter.info/jetstream/jetstream/index.html once deployed.